spec/jobs/generic_job_spec.rb

```diff
@@ -52,4 +52,83 @@ def perform(_bulk_action_id, _params)
       expect(bulk_action_no_process_callback.druid_count_total).to eq 1
     end
   end
+
+  describe '#open_new_version' do
+    let(:current_user) do
+      instance_double(User,
+                      is_admin?: true)
+    end
+
+    before do
+      @dor_object = double(pid: 'druid:123abc')
```

Comment:
Can you switch these to the `let()` syntax rather than instance variables? http://www.betterspecs.org/#let

Reply:
yes, will do